### DISCUSSION TITLE
Alert state if no data or all values are null

### DISCUSSION THREAD

**Discussion by afinzel** [grafana/grafana#55899] [Alerting]
I think in an older version of Grafana.  "Alert state if no data or all values are null" had an option not to change the alert state.  The problem is we have Grafana talking to AWS and if we set "Alert state if no data or all values are null" to OK it sometimes resolves issues which are not resolved.  If we set it to error, we get false alarms.  There should be an option no change which leaves the alert as is.

**Comment by jgournet** [comment]
Thanks ! this is exactly what I would need too !
I opened https://github.com/grafana/grafana/issues/33828 a long time ago, which was closed for reasons ... but none solved my issue so far.

Are we missing some option ? or can this be implemented ?

**Comment by jgournet** [comment]
found https://github.com/grafana/grafana/discussions/43588 which says we just need to silence nodata alerts ?
it seems like a sad workaround for a missing feature - I'll try it anyway

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
